src/pages/Dashboard/Dashboard.tsx

```diff
@@ -13,14 +13,21 @@ export default function Dashboard() {
     const navigateTo = useNavigate();
     const [isLoggedIn, setIsLoggedIn] = useState(false);
     const [memberId, setMemberId] = useState<string | undefined>(undefined);
+    const [displayName, setDisplayName] = useState<string | undefined>(undefined);
+    const [welcomeMessage, setWelcomeMessage] = useState<string>('Welcome guest');
 
     function checkMember() {
         try {
             const loggedIn = localStorage.getItem('member_id') !== null;
```

Comment:
You can check if a user is logged by calling 
```
// auth object from firebase.ts
auth.currentUser
```
which either returns the currently signed-in user, or null. The user object contains the uid, displayName, email, etc.